explorer/explorerroutes.go

```diff
@@ -1837,14 +1837,33 @@ func (exp *explorerUI) ProposalPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Proposals whose voting hasn't commenced do not have an end height assigned yet.
+	// Ignore the error returned since the endheight variable will be a zero value.
+	endHeight, _ := strconv.ParseInt(proposalInfo.ProposalVotes.Endheight, 10, 64)
+
+	var timeLeft string
+	var blocksLeft = endHeight - exp.Height()
+
+	if blocksLeft > 0 {
```

Comment:
Please don't feel like you need to change this, but one thing I'm trying to be more mindful of is limiting variable scope.  I might have considered something like:

```go
if blocksLeft := endHeight - exp.Height(); blocksLeft > 0 {
```